### ISSUE TITLE
Using Mandarin in slug will break profile page

### ISSUE THREAD

**Issue by MrOrz** [cofacts/rumors-site#399]
When Mandarin is used in slug, user will be redirected to `/user/<Mandarin>`, but will show "The user does not exist".

**Comment by MrOrz** [comment]
The root cause is that we did not update rumors-db schema on production to latest. Currently `slug` field mapping is automatically generated value:
```
          "slug" : {
            "type" : "text",
            "fields" : {
              "keyword" : {
                "type" : "keyword",
                "ignore_above" : 256
              }
            }
          },
```

To resolve, we must reindex to the latest mapping.

**Comment by MrOrz** [comment]
Fixed in cofacts/rumors-db#53, closing.
